scripts/deploy-backend.sh

```diff
@@ -42,7 +42,9 @@ set_cf_envs()
   "AV_SCAN_URL"
   "BASE_URL"
   "CLAMAV_NEEDED"
+  "CYPRESS_TOKEN"
   "DJANGO_CONFIGURATION"
+  "DJANGO_DEBUG"
```

Comment:
Were these added to CircleCI env vars? Having missing/empty env vars has given us problems in the past.

Reply:
CYPRESS_TOKEN is already in there, yes. It's actually being used to populate the CYPRESS_TOKEN below. DJANGO_DEBUG is also in there, but I'm not trying to grab it, instead just explicitly setting to "yes"